js/util/attachment.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/prefer-default-export */
+export function dataUrlToAttachment(dataUrl) {
+  if (!dataUrl || !dataUrl.length) {
+    return [];
+  }
+
+  const [propertiesString, filedata] = dataUrl.split(':', 2)[1].split(',');
```

Comment:
Does it make sense to have an example of what we expect the `dataUrl` to look like in case something ever changes with the library we're using and something needs to be troubleshot? (I guess maybe the tests do that?)